docs/changelog/99527.yaml

```diff
@@ -0,0 +1,5 @@
+pr: 99445
```

Comment:
should be 99527 - this creates a wrong PR reference link in the release notes.

Reply:
Fixing, https://github.com/elastic/elasticsearch/pull/101908 :D